src/ext/command.rs

```diff
@@ -639,6 +639,20 @@ impl Description {
             .join(", ");
         format!("[{}]", joined)
     }
+
+    /// Provides a representation of the command that can be pasted into the terminal
+    pub fn pastable(&self) -> String {
+        let envs = self
+            .envs
+            .clone()
+            .into_iter()
+            .filter(|e| !e.contains("<REMOVED>"))
```

Comment:
If we tell them to run this command, and they do, and it does dynamic analysis, they'll be potentially confused why we don't when it eventually runs in Broker.

What do you think of just stripping `<REMOVED>`, so that the command looks like `PATH='' fossa analyze ...`?

Reply:
Oh, interesting. I was only thinking of the case where we remove `GIT_ASKPASS`. I believe that fails if you set it to `''`.

Testing...

Nope, that works. Okay, that seems like a good plan. I'll make that change